mist/meson.build

```diff
@@ -0,0 +1,5 @@
+header_tgts = []
+
+foreach header : headers
+  header_tgts += custom_target('copy'+header, input: join_paths(meson.source_root(),'lib',header), output: header, command: ['cp', '@INPUT@', '@OUTPUT@'])
```

Comment:
If you used `files()` when initially defining the headers, you would not need `join_paths(meson.source_root(), 'lib', ...)` here.

Actually the next version of Meson makes this so much easier: https://mesonbuild.com/Fs-module.html#copyfile

```meson
fs = import('fs')

foreach header: headers
    header_tgts += fs.copyfile(header)
endforeach
```

Reply:
files wouldn't work here :/ At least not without fs.copyfile, since a file obj cannot be turned into a string.. :/

Reply:
Though tbh; I've always found projects that structure their headers differently than their installed layout very ugly and are sometimes problematic to work with ;) 

It would've been so much easier if there was an 'include/mist/' folder that actually housed the headers, then that could've just been added to the include path without any trickery